runtime/manual/tools/documentation_generator.md

```diff
@@ -25,6 +25,52 @@ function add(x: number, y: number): number
   Adds x and y. @param {number} x @param {number} y @returns {number} Sum of x and y
 ```
 
+## Linting
+
+You can use `--lint` flag to check for problems in your documentation.
+`deno doc` will point out missing JSDoc comments and return types for public
+APIs.
+
+```js
+// multiply.ts
```

Comment:
Might be best to rename this `mod.ts` so people get the idea to provide the entrypoint.

Reply:
also you can provide a title in code blocks if you want rather than a comment: https://docusaurus.io/docs/markdown-features/code-blocks#code-title

Reply:
Oh, cool!